be/src/vec/olap/block_reader.cpp

```diff
@@ -130,6 +131,7 @@ OLAPStatus BlockReader::init(const ReaderParams& read_params) {
                     _normal_columns_idx.emplace_back(j);
                 } else {
                     _agg_columns_idx.emplace_back(j);
+                    _real_columns_idx[j] = cid;
```

Comment:
I think it is better to named `_real_agg_columns_idx`  or `_origin_agg_columns_idx`